common/src/main/java/co/elastic/otel/common/ElasticAttributes.java

```diff
@@ -30,11 +30,11 @@ public interface ElasticAttributes {
   AttributeKey<String> SPAN_TYPE = AttributeKey.stringKey("elastic.span.type");
   AttributeKey<String> SPAN_SUBTYPE = AttributeKey.stringKey("elastic.span.subtype");
 
-  /** Marker attribute for inferred spans. */
-  AttributeKey<Boolean> IS_INFERRED = AttributeKey.booleanKey("elastic.is_inferred");
-
-  /** Used as marker on span-links to override the parent-child relationship for inferred spans. */
-  AttributeKey<Boolean> IS_CHILD = AttributeKey.booleanKey("elastic.is_child");
+  /**
+   * Marker attribute for inferred spans. Does not have the elastic-prefix anymore because it has
+   * been contributed upstream
+   */
+  AttributeKey<Boolean> IS_INFERRED = AttributeKey.booleanKey("is_inferred");
```

Comment:
[minor] do we have a way to reuse the upstream attribute without duplicating it ?

Reply:
Not here, because we don't want this package to pull in the inferred spans extension as a dependency.